### DISCUSSION TITLE
Welcome to sidepiece Discussions!

### DISCUSSION THREAD

**Discussion by electricvino** [electricvino/sidepiece#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by electricvino** [comment]
So here is what I was thinking for a project.  I am a sucker for YouTube productivity videos.  Makes me feel more productive just by sitting around.  Anyway I was watching https://youtu.be/6o2tm00Ar8A and about the 2 min mark he is looking at a spreadsheet to have a year long view of his calendar.  Thought that could be a fun app to do.  Use something like Tauri fo make it a desktop app and then integrate into google calendar.  UI could be fun with angular or svelte or whatever you guys wanted to do.  Let me know thoughts or other ideas.

**Reply by hshoemaker** [reply]
![Screenshot 2023-05-03 132016](https://user-images.githubusercontent.com/26283006/236031491-185c3ac0-070e-4cca-af48-3d1391bfb931.jpg)
So something like this?

**Reply by electricvino** [reply]
yeah or the spreadsheet he shows shortly after.  Watching the vid just kind of popped out as that might be an interesting project.  What do y'all think?

**Reply by hshoemaker** [reply]
I dig it! I like the idea of using Tauri as well. I've been wanting to do a project in that.

**Reply by RepositoryOfCalories** [reply]
Sounds fun! A feature that immediately came to mind is a toggle to switch between displaying the entire calendar year to showing the remaining months. The toggle would probably only be visible if the current month is greater than January or something.

**Reply by RepositoryOfCalories** [reply]
@electricvino @hshoemaker what platform do we wanna use to track units of work (like Trello, etc)?

**Reply by hshoemaker** [reply]
@RepositoryOfCalories I imagine we can use some of the Github features, like Projects and Issues

**Comment by hshoemaker** [comment]
So my thought is using:
[Svelte](https://svelte.dev/) for the web app 
[Tauri](https://tauri.app/) for the mobile app
[Buma](https://bulma.io/) or some other non-bootstrap css library

thoughts?

**Reply by electricvino** [reply]
I'll reload the files with svelte instead of angular and see if I can get GH to auto-build too.

**Comment by electricvino** [comment]
You can download sidepiece and run the template.  clone, npm install and you should be able to _npm run tauri dev_

if that does not work you may need to install dependencies like pnpm or vite.
